packages/services/api/src/modules/organization/providers/organization-members.ts

```diff
@@ -178,11 +188,12 @@ export class OrganizationMembers {
               `
             : sql``
         }
+        ${searching ? sql`AND ("u"."display_name" ILIKE ${'%' + searchTerm + '%'} OR "u"."email" ILIKE ${'%' + searchTerm + '%'})` : sql``}
       ORDER BY
         "om"."organization_id" DESC
+        , "om"."created_at" DESC
         , "om"."user_id" DESC
```

Comment:
How does this affect usage of existing indices?

Reply:
Now I get it, so in case of, we would need to update the index as well. 😢 Can be a follow-up PR, though!

```
 "om"."organization_id" DESC
        , "om"."created_at" DESC
        , "om"."user_id" DESC
```